### ISSUE TITLE
swagger-ui-dist version 3.18.3-republish is broken

### ISSUE THREAD

**Issue by rafalsiwiec** [swagger-api/swagger-ui#4863]
Loading of `swagger-ui-dist` fails like this:

```> require('swagger-ui-dist')
Error: Cannot find module './absolute-path.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
```

Looks like package https://registry.npmjs.org/swagger-ui-dist/-/swagger-ui-dist-3.18.3-republish.tgz is missing the `absolute-path.js`. Content of unpacked archive is:

```README.md
favicon-16x16.png
favicon-32x32.png
index.html
index.js
oauth2-redirect.html
package.json
swagger-ui-bundle.js
swagger-ui-bundle.js.map
swagger-ui-standalone-preset.js
swagger-ui-standalone-preset.js.map
swagger-ui.css
swagger-ui.css.map
swagger-ui.js
swagger-ui.js.map
```

**Comment by silverwind** [comment]
Whoever published `3.18.3-republish` should be aware that one can [unpublish](https://docs.npmjs.com/cli/unpublish) for up to 72 hours after the initial publishing, so there is no need to republish under a different name.

**Comment by shockey** [comment]
Ok - `swagger-ui-dist@3.18.3-republish2` has been published, and can be successfully required in Node.

Sorry for the inconvenience, everyone!
